src/content/articles/values-narrowed-before-usage.mdx

```diff
@@ -0,0 +1,112 @@
+---
+date: "February 17 2023"
+description: "Why TypeScript's type narrowing won't apply to ."
+meta: literals, narrowing, objects, primitive
+---
+
+# Why Values Aren't Narrowed After Function Calls
+
+TypeScript's type system boasts a feature called _type narrowing_.
+Type narrowing means that if a block of code can only be reached when a value is a specific type, then the value will be considered that type inside the block.
+
+As an example, take the following `fruit` variable of type `string | undefined` (a union type indicating it can be either `string` or `undefined`.
```

Comment:
[Proofreading] Seems similar to this part [in the other article](https://github.com/LearningTypeScript/site/pull/31/files#r1276719547). 
```suggestion
For example, take the following `fruit` variable of type `string | undefined` (a union type indicating it can be either `string` or `undefined`.
```